src/app/forms/services/dynamic-form.service.ts

```diff
@@ -18,7 +19,7 @@ type CustomFormFields = CustomFormControl | CustomFormArray | CustomFormGroup;
   providedIn: 'root'
 })
 export class DynamicFormService {
-  constructor(private store: Store<State>) {}
+  constructor(private store: Store<State>, private techRecordService: TechnicalRecordService) {}
```

Comment:
PART 3: 
And then remove the dependency on the techRecordService from the dynamic form group
```suggestion
  constructor(private store: Store<State>) {}
```